pkg/infra/metrics/metrics.go

```diff
@@ -175,6 +175,9 @@ var (
 	// StatsTotalAnnotations is a metric of total number of annotations stored in Grafana.
 	StatsTotalAnnotations prometheus.Gauge
 
+	// StatsTotalAlertRules is a metric of total number of alert rules stored in Grafana.
+	StatsTotalAlertRules prometheus.Gauge
+
```

Comment:
@kylebrandt I was just adding the same for library panels and variables by copying this PR, but I think you might need to register this here too:
https://github.com/grafana/grafana/blob/60d0c8d0ec7df02e7348f62791caeeaa03a59eee/pkg/infra/metrics/metrics.go#L587

I couldn't get the metrics to show up on `http://localhost:3000/metrics` otherwise